lib/connect.js

```diff
@@ -158,10 +159,6 @@ var KiteConnect = function(params) {
         }
```

Comment:
`paramsSerializer` should be removed since now you do serialization based on flag sent.

Reply:
Done.